**Additional resources supporting the PhD thesis of Simran Krishnakant Kushwaha**

**Chapter 2-**

•	An interactive visualisation highlighting 26 identified spots in _Salmonella_ RefSeq.

Spot 1 – A hotspot for defence systems, particularly Septu type I 

Spot 9 – A hotspot for virulence factors, particularly _sod_ 

Spot 11 – A hotspot for defence systems, particularly AbiE and Shango 

Spot 15 – A hotspot for virulence factors, particularly _cdt_ 

Spot 17 – A hotspot for stress-resistance genes against gold 

Spot 21 – A hotspot for virulence factors _ste_ and _sse_ 

Spot 22 – A hotspot for defence systems CRISPR-Cas type I-E 

Spot 30 – A hotspot for virulence factors _lpf_ 

Spot 31 – A hotspot for defence systems SEFIR 

Spot 32 – A hotspot for virulence factors _sse_ 

Spot 36 – A hotspot for virulence factors, particularly _fae_ 

Spot 39 – A hotspot for multiple defence systems 

Spot 43 – A hotspot for multiple defence systems 

Spot 44 – A hotspot for multiple antibiotic resistance genes 

Spot 47 – A hotspot for virulence factor _tcp_ and defence systems Thoeris type I and II 

Spot 51 – A hotspot for multiple antibiotic and stress-resistance genes 

Spot 53 – A hotspot for stress-resistance genes against copper and silver 

Spot 54 – A hotspot for multiple virulence factors 

Spot 63 – A hotspot for multiple defence systems 

Spot 66 – A hotspot for defence system CBASS type I 

Spot 68 – A hotspot for defence system RM type I and IV 

Spot 79 – A hotspot for virulence factor _ssp_ 

Spot 89 – A hotspot for defence system RM type III 

Spot 92 – A hotspot for virulence factor _sop_ 

Spot 94 – A hotspot for virulence factor _rat_ 

Spot 103 – A hotspot for stress-resistance genes against arsenic

•	Microreact project 1, which presents metadata for isolates, phylogenetic analysis, and the country of isolation of _Salmonella_ RefSeq.

•	Microreact project 2, illustrating the serovar-wise distribution of pathogenic determinants on _Salmonella_ RefSeq.

•	Supplementary tables 2.1 to 2.13-

S2.1. Features of the 12,244 _Salmonella_ genomes analysed in this study.

S2.2. Features of the plasmids found across _Salmonella_ RefSeq.

S2.3. Prevalence of plasmid incompatibility groups across _Salmonella_ RefSeq.

S2.4. Features of the prophages found across _Salmonella_ RefSeq.

S2.5. List of pathogenicity genes analysed in this study.

S2.6. Distribution and location of virulence factors, antibiotic resistance genes, stress resistance genes, and defence systems in _Salmonella_ RefSeq.

S2.7. Average of plasmids, prophages, virulence factors, antibiotic resistance genes, stress resistance genes, and defence systems in _Salmonella_ RefSeq.

S2.8. Prevalence (%) of virulence factors, antibiotic resistance genes, stress resistance genes, and defence systems across plasmid incompatibility groups.

S2.9. Core, persistent, shell and cloud genes identified in _Salmonella_, and their predicted function.

S2.10. Regions of genomic plasticity identified in in _Salmonella_ RefSeq.

S2.11. RGP families and gene count in _Salmonella_ spots.

S2.12. Virulence factors, antibiotic resistance genes, stress resistance genes, and defence systems present on spots of integration in _Salmonella_.

S2.13. Flanking genes defining the integration spot and their predicted function.


**Chapter 3-**

•	The spacer conservation of 133 strains belonging to 26 serovars.

•	The supplementary table provides a detailed list of all strains utilised in the study.


**Chapter 4-**

•	Interactive network showcasing the interactions between potential regulatory spacers and their gene targets for _Salmonella_ _enterica_ serovar Typhi, Typhimurium and Enteritidis.
 
