# Exploring-and-Exploiting-Prokaryotic-Immunity-in-Salmonella

Additional resources supporting the PhD thesis

**Chapter 2-**

•	An interactive visualisation highlighting 26 identified spots in Salmonella RefSeq.

Spot 1 – A hotspot for defence systems, particularly Septu type I 

Spot 9 – A hotspot for virulence factors, particularly sod 

Spot 11 – A hotspot for defence systems, particularly AbiE and Shango 

Spot 15 – A hotspot for virulence factors, particularly cdt 

Spot 17 – A hotspot for stress-resistance genes against gold 

Spot 21 – A hotspot for virulence factors ste and sse 

Spot 22 – A hotspot for defence systems CRISPR-Cas type I-E 

Spot 30 – A hotspot for virulence factors lpf 

Spot 31 – A hotspot for defence systems SEFIR 

Spot 32 – A hotspot for virulence factors sse 

Spot 36 – A hotspot for virulence factors, particularly fae 

Spot 39 – A hotspot for multiple defence systems 

Spot 43 – A hotspot for multiple defence systems 

Spot 44 – A hotspot for multiple antibiotic resistance genes 

Spot 47 – A hotspot for virulence factor tcp and defence systems Thoeris type I and II 

Spot 51 – A hotspot for multiple antibiotic and stress-resistance genes 

Spot 53 – A hotspot for stress-resistance genes against copper and silver 

Spot 54 – A hotspot for multiple virulence factors 

Spot 63 – A hotspot for multiple defence systems 

Spot 66 – A hotspot for defence system CBASS type I 

Spot 68 – A hotspot for defence system RM type I and IV 

Spot 79 – A hotspot for virulence factor ssp 

Spot 89 – A hotspot for defence system RM type III 

Spot 92 – A hotspot for virulence factor sop 

Spot 94 – A hotspot for virulence factor rat 

Spot 103 – A hotspot for stress-resistance genes against arsenic

•	Microreact project 1, which presents metadata for isolates, phylogenetic analysis, and the country of isolation of Salmonella RefSeq.

•	Microreact project 2, illustrating the serovar-wise distribution of pathogenic determinants on Salmonella RefSeq.

•	Supplementary tables 2.1 to 2.13-

S2.1. Features of the 12,244 Salmonella genomes analysed in this study.

S2.2. Features of the plasmids found across Salmonella RefSeq.

S2.3. Prevalence of plasmid incompatibility groups across Salmonella RefSeq.

S2.4. Features of the prophages found across Salmonella RefSeq.

S2.5. List of pathogenicity genes analysed in this study.

S2.6. Distribution and location of virulence factors, antibiotic resistance genes, stress resistance genes, and defence systems in Salmonella RefSeq.

S2.7. Average of plasmids, prophages, virulence factors, antibiotic resistance genes, stress resistance genes, and defence systems in Salmonella RefSeq.

S2.8. Prevalence (%) of virulence factors, antibiotic resistance genes, stress resistance genes, and defence systems across plasmid incompatibility groups.

S2.9. Core, persistent, shell and cloud genes identified in Salmonella, and their predicted function.

S2.10. Regions of genomic plasticity identified in in Salmonella RefSeq.

S2.11. RGP families and gene count in Salmonella spots.

S2.12. Virulence factors, antibiotic resistance genes, stress resistance genes, and defence systems present on spots of integration in Salmonella.

S2.13. Flanking genes defining the integration spot and their predicted function.


**Chapter 3-**

•	The spacer conservation of 133 strains belonging to 26 serovars.

•	The supplementary table provides a detailed list of all strains utilised in the study.


**Chapter 4-**

•	Interactive network showcasing the interactions between potential regulatory spacers and their gene targets for Salmonella enterica serovar Typhi, Typhimurium and Enteritidis.
 
